hooks/_common.sh

```diff
@@ -1,6 +1,22 @@
 #!/usr/bin/env bash
 set -eo pipefail
 
+if [[ $PCT_LOG == trace ]]; then
+
+  echo "BASH path: '$BASH'"
+  echo "BASH_VERSION: $BASH_VERSION"
+  echo "BASHOPTS: $BASHOPTS"
+  echo "OSTYPE: $OSTYPE"
+
+  # ${FUNCNAME[*]} - reverse functions trace. Each new function call append to start
```

Comment:
```suggestion
  # ${FUNCNAME[*]} - reverse functions trace. Each new function call is appended to the beginning
```
I also don't quite understand what "_reverse functions trace_" means 😞

Reply:
`trace: _getopt_parse _getopt_main getopt common::parse_cmdline main main`

plain bash called function main
which called main
which called common::parse_cmdline
which called getopt
which called _getopt_main
which called _getopt_parse


Or in other words it can be visualized as:

`trace: _getopt_parse <- _getopt_main <- getopt <- common::parse_cmdline <- main <- main`

But I not sure how to visualize it this construction, so I left spaces

Reply:
"_function calls in reversed order_"?